doc/nrf/ug_thread_configuring.rst

```diff
@@ -477,17 +477,23 @@ Updating pre-built OpenThread libraries
 You can update the :ref:`nrfxlib:ot_libs` in nrfxlib when using any Thread sample if you configure the sample to build the OpenThread stack from source with :kconfig:option:`CONFIG_OPENTHREAD_SOURCES`.
 Use this functionality for :ref:`certification <ug_thread_cert>` of your configuration of the OpenThread libraries, for example.
 
+You can install the libraries either with or without debug symbols.
+Installing the libraries with debug symbols can be useful when debugging, but will take a significant amount of storage memory.
+Take this into account while committing those libraries to the repository.
```

Comment:
Not sure if this sentence is very helpful. Customers are not intended to commit to the repository. The main goal for giving the opportunity to build and install the libraries in the nrfxlib repo is actually for us to provide updated libraries. The secondary goal is for the customers to build their customized libraries, which won't ever be pushed to the nrfxlib repo, but rather to some (private) fork.